index.html

```diff
@@ -3594,6 +3594,54 @@ <h2>Revision History</h2>
 
     <section class="informative">
       <h2>Acknowledgements</h2>
+
+      <p>
+Work on this specification has been supported by the Rebooting the
+Web of Trust community facilitated by Christopher Allen, Shannon Appelcline,
+Kiara Robles, Brian Weller, Betty Dhamers, Kaliya Young, Manu Sporny,
+Drummond Reed, Joe Andrieu, Heather Vescent, Kim Hamilton Duffy, Samantha Chase,
+and Andrew Hughes. The participants in the Internet Identity Workshop,
+facilitated by Phil Windley, Kaliya Young, Doc Searls, and Heidi Nobantu Saul,
+also supported the refinement of this work through numerous working sessions
+designed to educate about, debate on, and improve this specification.
+      </p>
+
+      <p>
+The Working Group also thanks our Chairs, Brent Zundel and Kristina Yasuda, as
+well as our W3C Staff Contact, Ivan Herman, for his expert management and steady
+guidance of the group through the W3C standardization process.
+      </p>
+
+      <p>
+Portions of the work on this specification have been funded by the United States
+Department of Homeland Security's Science and Technology Directorate under
+contracts 70RSAT20T00000029, 70RSAT21T00000016, and 70RSAT23T00000005. The
+content of this specification does not necessarily reflect the position or the
+policy of the U.S. Government and no official endorsement should be inferred.
+      </p>
+
+      <p>
+The Working Group would like to thank the following individuals for reviewing
+and providing feedback on the specification (in alphabetical order):
+      </p>
+
+      <p>
+Will Abramson, Mahmoud Alkhraishi, Christopher Allen, Joe Andrieu, Bohdan
+Andriyiv, Anthony, George Aristy, Greg Bernstein, Bob420, Sarven Capadisli,
+Melvin Carvalho, David Chadwick, Matt Collier, Gabe Cohen, Sebastian Crane, Kim
+Hamilton Duffy, Snorre Lothar von Gohren Edwin, Veikko Eeva, Eric Elliott, Filip
+Kolarik, Raphael Flechtner, Julien Fraichot,  Benjamin Goering, Kyle Den Hartog,
+Joseph Heenan, Helge, Ivan Herman, Michael Herman, Anil John, Andrew Jones,
+Michael B. Jones, Ted Thibodeau Jr, Gregory K, Gregg Kellogg, David I. Lehn,
+Charles E. Lehner, Christine Lemmer-Webber, Eric Lim, Dave Longley, Tobias
+Looker, nightpool, Jer Miller, Bert Van Nuffelen, Luis Osta, Nate Otto, George
+J. Padayatti, Addison Phillips, Mike Prorock, Brian Richter, Rieks Joosten,
+Anders Rundgren, Eugeniu Rusu, Markus Sabadello, silverpill, Wesley Smith, Manu
+Sporny, Patrick St-Louis, Orie Steele, Henry Story, Oliver Terbu, John Toohey,
+Mike Varley, Jeffrey Yasskin, Kristina Yasuda, Benjamin Young, Dmitri Zagidulin,
+and Brent Zundel
```

Comment:
Alphabetical name ordering is more challenging than it may appear... This reformatting of HTML should make future revisions easier. Persons listed by a single name/handle are now ordered as if that is their surname. Best effort has been made to appropriately sort persons who have more than 2 elements (forename, surname) in their name (suffix, nobiliary particle, etc.). Corrections welcome, especially directly by the named persons!
```suggestion
Will Abramson,
Mahmoud Alkhraishi,
Christopher Allen,
Joe Andrieu,
Bohdan Andriyiv,
Anthony,
George Aristy,
Greg Bernstein,
Bob420,
Sarven Capadisli,
Melvin Carvalho,
David Chadwick,
Matt Collier,
Gabe Cohen,
Sebastian Crane,
Kyle Den Hartog,
Veikko Eeva,
Eric Elliott,
Filip Kolarik,
Raphael Flechtner,
Julien Fraichot,
Benjamin Goering,
Kim Hamilton Duffy, 
Joseph Heenan,
Helge,
Ivan Herman,
Michael Herman,
Anil John,
Andrew Jones,
Michael B. Jones,
Rieks Joosten,
Gregory K,
Gregg Kellogg,
David I. Lehn,
Charles E. Lehner,
Christine Lemmer-Webber,
Eric Lim,
Dave Longley,
Tobias Looker,
Jer Miller,
nightpool,
Luis Osta,
Nate Otto,
George J. Padayatti,
Addison Phillips,
Mike Prorock,
Brian Richter,
Anders Rundgren,
Eugeniu Rusu,
Markus Sabadello,
silverpill,
Wesley Smith,
Manu Sporny,
Patrick St-Louis,
Orie Steele,
Henry Story,
Oliver Terbu,
Ted Thibodeau Jr,
John Toohey,
Bert Van Nuffelen,
Mike Varley,
Snorre Lothar von Gohren Edwin,
Jeffrey Yasskin,
Kristina Yasuda,
Benjamin Young,
Dmitri Zagidulin,
and
Brent Zundel.
```

Reply:
@TallTed sharing feedback on the sorting algorithm, look at (my ;) name: 'Filip Kolarik', Kolarik is a family name, and Filip is the first name.  It should be placed after 'Gregg Kellogg'. Anyway, thank you for the acknowledgement.

A note to language enthusiasts: Czech language is phonetic, you write as you hear, therefore 'F' instead of PH.

Reply:
@filip26 -- Thanks for the catch! I accidentally left you in the original sort order. Fixed in #303.